src/components/plots/rectanglePlot.ts

```diff
@@ -61,6 +82,18 @@ export module Plots {
       return [{attrToProjector: this._generateAttrToProjector(), animator: this._getAnimator("rectangles")}];
     }
 
+    public x(): AccessorScaleBinding<X, number>;
+    public x(x: number | Accessor<number>): Plots.Rectangle<X, Y>;
+    public x(x: X | Accessor<X>, scale: Scale<X, number>): Plots.Rectangle<X, Y>;
+    public x(x?: number | Accessor<number> | X | Accessor<X>, scale?: Scale<X, number>): any {
+      if (x == null) {
+        return this._propertyBindings.get(Rectangle._X_KEY);
+      }
+      this._bindProperty(Rectangle._X_KEY, x, scale);
+      this.renderImmediately();
+      return this;
+    }
```

Comment:
Because `Rectangle` extends off of `XYPlot` and adds the `x1`/`y1` API points, chaining no longer functions properly if we use the original `x`/`y` implementation from `XYPlot`